python/tvm/contrib/hexagon/pytest_plugin.py

```diff
@@ -260,9 +261,21 @@ def aot_target(aot_host_target):
         assert False, "Incorrect AoT host target: {aot_host_target}. Options are [c, llvm]."
 
 
+@pytest.fixture(scope="session")
+def skip_rpc(request) -> bool:
+    return request.config.getoption("--skip-rpc")
+
+
 def pytest_addoption(parser):
     parser.addoption("--gtest_args", action="store", default="")
 
+    parser.addoption(
+        "--skip-rpc",
+        action="store_true",
+        default=False,
+        help="If set true, a new hexagon launcher is not created",
```

Comment:
You can make it more clear by changing to this:
If set true, the RPC server initialization on Android would be skipped.

Reply:
Thanks. Done.